### DISCUSSION TITLE
CORS Error on GitHub Pages

### DISCUSSION THREAD

**Discussion by Timschmid** [community/community#36117] [Other Feature Feedback, Questions, & Ideas]
Hey, 
I have an angular Website, which is hosted on a Private GitHub Pages Site, So you have to be a member of the Repository to access the site. The Site is an PWA, so it will also be stored in the Local Cache. My Problem is now, when I Update my Angular Site and push it to the GitHub Repository, I've got Cor's Error, when I reload the Webpage and the new Update of the Website won't be shown.

![image](https://user-images.githubusercontent.com/22547772/195810814-2f08e121-12b9-4667-a6c9-26a0ff62870d.png)

I guess The problem is, that it tries to check, if I'm enabled to access the site. On Github.com/login... But the Website itself is at the domain .github.io. Does anybody know how to handle this?

**Comment by yoannchaudet** [comment]
I am not sure about your scenario here but given your site is private, authentication happens via classic redirections. When fetching a static assets (e.g. here a JSON file), the redirection cannot happen and you will likely want to implement client side logic to refresh the whole page so the authentication can proceed.

**Comment by Timschmid** [comment]
hmm I can try that thx
